code/controllers/subsystems/responseteam.dm

```diff
@@ -2,123 +2,92 @@
-	wait = 1800 //Fire only every 3 minutes - Not more often needed for now
+	flags = SS_NO_FIRE
 
-	var/progression_chance = 0
-	var/percentage_antagonists = 0
-	var/percentage_dead = 0
-	var/can_call_ert = TRUE
-	var/ert_type = "NT-ERT" //what ert type will be deployed
-	var/send_emergency_team = 0
 	var/ert_count = 0
+	var/send_emergency_team = FALSE
+	var/can_call_ert = TRUE
 
-	var/pg_green = 0
-	var/pg_yellow = 0
-	var/pg_red = 0
-	var/pg_delta = 0
+	var/list/datum/responseteam/available_teams = list()
+	var/datum/responseteam/picked_team
+	var/list/datum/ghostspawner/human/ert/sent_teams = list()
 
 /datum/controller/subsystem/responseteam/Recover()
-	progression_chance = SSresponseteam.progression_chance
-	can_call_ert = SSresponseteam.can_call_ert
-	ert_type = SSresponseteam.ert_type
 	send_emergency_team = SSresponseteam.send_emergency_team
-	pg_green = SSresponseteam.pg_green
-	pg_yellow = SSresponseteam.pg_yellow
-	pg_red = SSresponseteam.pg_red
-	pg_delta = SSresponseteam.pg_delta
 
 /datum/controller/subsystem/responseteam/New()
 	NEW_SS_GLOBAL(SSresponseteam)
 	feedback_set("responseteam_count",0)
 
+/datum/controller/subsystem/responseteam/Initialize(start_timeofday)
+	. = ..()
+	var/list/all_teams = subtypesof(/datum/responseteam)
+	for(var/team in all_teams)
+		CHECK_TICK
+		var/datum/responseteam/ert = new team
+		available_teams += ert
+
 /datum/controller/subsystem/responseteam/stat_entry()
-	var/out = "PGC:[progression_chance] "
-	out += "BC:[config.ert_base_chance]\n"
-	out += "PA:[percentage_antagonists] "
-	out += "PAF:[config.ert_scaling_factor_antag]\n"
-	out += "PD:[percentage_dead] "
-	out += "PDF:[config.ert_scaling_factor_dead]\n"
-	out += "SF:[config.ert_scaling_factor] "
-	out += "CC:[can_call_ert] "
+	var/out = "CC:[can_call_ert]"
 	..(out)
 
-/datum/controller/subsystem/responseteam/fire()
-	if(send_emergency_team == 0) // There is no ERT at the time.
-		var/total = 0
-		var/deadcount = 0
-		var/antagonists = 0
-		for(var/mob/living/carbon/human/H in mob_list)
-			if(!H.isMonkey() || H.client) 
-				total++
-				if(is_special_character(H) >= 1) antagonists++
-				if(H.stat == 2) deadcount++
-				
-		if(total == 0)
-			percentage_antagonists = 0
-			percentage_dead = 0
-		else
-			percentage_antagonists = round(100 * antagonists / total)
-			percentage_dead = round(100 * deadcount / total)
-		
-
-		switch(get_security_level())
-			if("green")
-				progression_chance += config.ert_green_inc
-				pg_green += config.ert_green_inc
-			if("blue")
-				progression_chance += config.ert_yellow_inc
-				pg_yellow += config.ert_yellow_inc
-			if("red")
-				progression_chance += config.ert_blue_inc
-				pg_red += config.ert_red_inc
-			if("delta")
-				progression_chance += config.ert_delta_inc
-				pg_delta += config.ert_delta_inc
-			else
-				progression_chance += 1
-
-/datum/controller/subsystem/responseteam/proc/trigger_armed_response_team(var/forced_choice = FALSE)
-	if(!can_call_ert && !forced_choice)
```

Comment:
By removing `&& !forced_choice` you removed the ability for admins to force a second specific response team (and automatically alert the crew with the default announcement)

Reply:
Solved